### PR TITLE
docker-network: add page

### LINKED ISSUE #4616: page request: docker-network
I would like to give it a go if issue is valid 🙂 
Official document: https://docs.docker.com/engine/reference/commandline/network/

### PR DESCRIPTION
**Pull request by Gau02**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Fixes #4616

### CODE REVIEW

File: pages/common/docker-network.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# docker network
+
+> Create and manage docker network.
+> More information: <https://docs.docker.com/engine/reference/commandline/network/>.
+
+- List all available and configured networks on docker daemon:
+
+`docker network ls`
+
+- Create user defined network:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:10]: Why `user defined`?
```suggestion
- Create a network:
```

File: pages/common/docker-network.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker network create --driver {{driver_name}} {{network_name}}`
+
+- Display detailed information on one or more networks:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:14]:
```suggestion
- Display detailed information of a space-separated list of networks:
```

File: pages/common/docker-network.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker network inspect {{network_name}}`
+
+- Connect a container to a network:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:18]: Based on `docker-inspect` page:
```suggestion
- Connect a container to a network using a name or ID:
```

File: pages/common/docker-network.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker network connect {{network_name}} {{container_name/id}}`
```
Comment (navarroaxel) [inline pages/common/docker-network.md:20]:
```suggestion
`docker network connect {{network_name}} {{container_name|ID}}`
```

File: pages/common/docker-network.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Disconnect a container from a network:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:22]:
```suggestion
- Disconnect a container from a network using a name or ID:
```

File: pages/common/docker-network.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker network disconnect {{network_name}} {{container_name/id}}`
```
Comment (navarroaxel) [inline pages/common/docker-network.md:24]:
```suggestion
`docker network disconnect {{network_name}} {{container_name|ID}}`
```

File: pages/common/docker-network.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Remove one or more networks in docker daemon:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:26]: Should be disconnected from any container before `rm`?
```suggestion
- Remove a space-separated list of networks:
```

File: pages/common/docker-network.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`docker network rm {{network_name}}`
+
+- Remove all unused(not referenced by any container) networks:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:30]:
```suggestion
- Remove all unused (not referenced by any container) networks:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @Gau02, welcome and thanks for your submission. I left you a comments.

File: pages/common/docker-network.md (hunk 1 above)
Comment (Gau02) [reply pages/common/docker-network.md:10]: @navarroaxel, `user defined` is because docker comes with a default bridge network as well.

Comment (navarroaxel) [reply pages/common/docker-network.md:10]: Ok @Gau02, then:
```suggestion
- Create a user defined network:
```

File: pages/common/docker-network.md (hunk 7 above)
Comment (Gau02) [reply pages/common/docker-network.md:26]: Yes. Should be disconnected from container before `rm` but docker gives you this detail if you try to remove network which is connected to containers and does not allow to remove it.

@navarroaxel, How about mentioning ` Remove one or more space-separated networks in docker daemon`?

Comment (navarroaxel) [reply pages/common/docker-network.md:26]: Based on `docker` page, the `docker rm` example. Maybe you can move the `prune` example above this and just say:
```suggestion
- Remove a space-separated list of unused networks:
```

File: pages/common/docker-network.md (hunk 5 above)
Comment (Gau02) [reply pages/common/docker-network.md:22]: @navarroaxel, I guess `using a name or ID` is not required as it is a default understanding for each command. but, if you require it, we need to add it after each command information as each command has operation on network.

File: pages/common/docker-network.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+- Disconnect a container from a network:
+
+`docker network disconnect {{network_name}} {{container_name|ID}}`
+
+- Remove a space-separated list of unused(not referenced with any container) networks:
+
+`docker network prune`
+
+- Remove a space-separated list of networks:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:30]:
```suggestion
- Remove a space-separated list of unused networks:
```

File: pages/common/docker-network.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Connect a container to a network using a name or ID:
+
+`docker network connect {{network_name}} {{container_name|ID}}`
+
+- Disconnect a container from a network:
+
+`docker network disconnect {{network_name}} {{container_name|ID}}`
+
+- Remove a space-separated list of unused(not referenced with any container) networks:
```
Comment (navarroaxel) [inline pages/common/docker-network.md:26]: :thinking:  I think `by` is the correct connector:
```suggestion
- Remove a space-separated list of unused (not referenced by any container) networks:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Gau02! I've just made 1 more teensy grammar fix I noticed.

**Pull request merged**